core/src/modules/configLoaders/property.ts

```diff
@@ -12,6 +12,15 @@ import { Op } from "sequelize";
 
 import { ConfigWriter } from "../configWriter";
 
+async function isBootstrappedProperty(
+  configObject: ConfigurationObject,
+  source: Source
+) {
+  const sourceMapping = await source.getMapping();
+  const mappedPropertyKeys = Object.values(sourceMapping);
+  return mappedPropertyKeys.includes(configObject.key || configObject.name);
+}
+
```

Comment:
Can you use `property.directlyMapped` here?

Reply:
you're right!